packages/measure-tools/src/api/MeasureToolsLoggerCategory.ts

```diff
@@ -11,6 +11,7 @@ export enum MeasureToolsLoggerCategory {
   Root = "Measure-Tools",
 }
 
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
where is this being redeclared?

Reply:
That's due to the namespace and enum. I really don't understand why both exist but thought that'd be a different PR, and later problem to fix.